TAG

```diff
@@ -1 +1 @@
-v0.49.0
+v1.0.0-beta.3
```

Comment:
once  we merge this will do a v1 release or do we want to do them together?

Reply:
It wont release nothing, as there is already v1.0.0-beta.3 and I didn't changed from there :)

The release v1 will be made after fixing the other small issues and Go version (hopefuly this monday)